report_thesis/src/sections/introduction.tex

```diff
@@ -5,30 +5,42 @@ \section{Introduction}\label{sec:introduction}
 Part of this research is facilitated through interpretation of spectral data gathered by \gls{libs} instruments, which fire a high-powered laser at soil samples to create a plasma.
 The emitted light is captured by spectrometers and analyzed using machine learning models to assess the presence and concentration of certain major oxides, informing NASA's understanding of Mars' geology.
 
-However, predicting major oxide compositions from \gls{libs} data presents significant computational challenges, including the high dimensionality and non-linearity of the data, compounded by multicollinearity and matrix effects~\cite{andersonImprovedAccuracyQuantitative2017}. 
-These effects can cause the intensity of emission lines from an element to vary independently of that element's concentration, introducing unknown variables that complicate the analysis. 
-Furthermore, due to the high cost of data collection, datasets are often small, which further complicates the task of building accurate and robust models.
+However, predicting major oxide compositions from \gls{libs} data still presents significant computational challenges.
+These include the high dimensionality and non-linearity of the data, compounded by issues of multicollinearity and matrix effects~\cite{andersonImprovedAccuracyQuantitative2017}.
+Such effects can cause the intensity of emission lines from an element to vary independently of that element's concentration, introducing unknown variables that complicate the analysis.
+Furthermore, the high cost of data collection often results in small datasets, exacerbating the difficulty of building accurate and robust models.
 
-Various machine learning models have been used to predict the composition of major oxides in the sample, including \glspl{cnn}~\cite{yang_laser-induced_2022, yangConvolutionalNeuralNetwork2022}, \gls{svr}~\cite{rezaei_dimensionality_reduction}, and hybrid models like \gls{df}-\gls{k-elm}~\cite{song_DF-K-ELM} that incorporate domain knowledge to enhance model interpretability and performance.
-However, the high dimensionality and multicollinearity of the spectral data remains a significant challenge for these models.
+Previous work has aimed to improve the prediction of major oxide compositions from LIBS data.
+Regression techniques and dimensionality reduction with feature selection have been used to enhance the accuracy and interpretability of these models.
+Tailored approaches have also been developed, where different models are selected based on their performance with specific spectral characteristics and distances~cite{rezaei_dimensionality_reduction, andersonPostlandingMajorElement2022}.
```

Comment:
```suggestion
Tailored approaches have also been developed, where different models are selected based on their performance with specific spectral characteristics and distances~\cite{rezaei_dimensionality_reduction, andersonPostlandingMajorElement2022}.
```

I don't think 'distances' make sense to readers here. Not readily apparent.

Reply:
Removed 'distances' since 'spectral characteristics' also encapsulates the distance (since the characteristics change based on the distance)